test/test_example.py

```diff
@@ -12,6 +12,8 @@
 
 @genty
 class ExampleTests(TestCase):
+    _multiprocess_can_split_ = True
```

Comment:
It tells nose's multi-processing logic that this single test class can be "split" across multiple processes.